networking.md

```diff
@@ -0,0 +1,43 @@
+# Networking
+
+Each machine (co-signing server, watchtower, wallet client) requires the capacity to connect to the others over TCP (mediated by the sync server operating a SOCKS5 proxy), and to engage in a noise_kk (or noise_xk) handshake (using pre-shared static public keys) to establish an authenticated and encrypted channel. 
```

Comment:
Could you clarify "mediated by the sync server operating a SOCKS5 proxy" ? I expect all entities to run `tor`.